FoundationsMastery/expected_length.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace FoundationsMastery
+{
+    [Serializable]
+    internal class expected_length : Exception
```

Comment:
Looks like Visual Studio generated this class to get rid of build errors... we don't need it!